src/org/labkey/test/tests/FileBasedPipelineTest.java

```diff
@@ -72,6 +73,28 @@ public void startTest()
         goToProjectHome();
     }
 
+    @Test
+    public void testScriptBasedFilewatchers()
+    {
+        String pipelineTaskWithHelpText = "Pipeline task with help text";
+        String helpText = "Hello! I am help text.";
+        String pipelineTaskNoHelpText = "two-steps";
+        String pipelineTaskInvisible = "Task that is not evokable by filewatcher";
+
+        goToFolderManagement().goToImportTab();
+        clickAndWait(Locator.lkButton("Manage file watcher triggers"));
+        DataRegionTable pipelineTable = new DataRegionTable("query", getDriver());
+        pipelineTable.clickInsertNewRow();
+
+        selectOptionByText(Locator.id("pipelineTaskSelect"), pipelineTaskWithHelpText);
+        Locator.byClass("lk-trigger-help-text").containing(helpText);
+
+        Locator.id("pipelineTaskSelect").notContaining(pipelineTaskInvisible);
```

Comment:
What is your intent with lines 92 and 95?  You can use these locators to .findElement(scope), or to get a boolean with .existsIn(scope)- but these lines appear to declare the locators without either assigning them to a variable or using it to validate.

Reply:
Oh, I think I was mistaken on what notContaining() did!
I wanted to check that the pipelineTaskInvisible and the 'null' did not exist within those specific elements. Should I be using existsIn() for that check?